lang/cs/Org.Apache.REEF.Common.Tests/metrics/MetricTestUtils.cs

```diff
@@ -104,6 +105,16 @@ public IMetricsCollector EndRecord()
                 throw new NotImplementedException();
             }
 
+            public IMetricsRecord GetRecord()
+            {
+                throw new NotImplementedException();
```

Comment:
@tcNickolas Test utilities like these are used to test other classes and not this utility class itself. Not implementing some of these functions mean that other classes that we are testing never call functions from this class and hence these function should not be implemented.